src/fields/Fields.cpp

```diff
@@ -695,6 +695,40 @@ Fields::InterpolateFromLev0toLev1 (amrex::Vector<amrex::Geometry> const& geom, c
 }
 
 
+void
+Fields::LevelUp (amrex::Vector<amrex::Geometry> const& geom, const int lev,
+                 const int which_slice, const std::string component)
+{
+    if (lev == 0) return; // only interpolate field to lev 1
+    HIPACE_PROFILE("Fields::LevelUp()");
+    constexpr int interp_order = 2;
```

Comment:
A `interp_order` stencil was already applied from beam particles to the grid. this is an additional smoothing step that is unnecessary, in particular because we are interpolating to a fine grid to resolve finer structures. A lower interpolation order here should reduce the difference with `jx_beam` obtained by depositing directly to the fine grid.
```suggestion
    constexpr int interp_order = 1;
```

Reply:
Currently all interpolation to level one is done with order two. Should the other functions be changed as well or just this one.

Reply:
Probably all of them, but we should review this case by case (might sometimes depend on whether it's a source or a field). But you're right, it's not only about this one. Could you open a separate PR setting all others to e.g. 1 (or have a variable controlling them, if that's quick, but we'll probably scan it just in the next few days/weeks before merging the PR and set it to the best value)? Let me know if that works for you, and if so we can merge this PR as is for consistency.

Reply:
Yes, it is easier to change and test everything in one go after merging this PR with order two.

Reply:
Let me take it back: we don't need to scan it. I believe that hard-coding it to 1 everywhere is what makes most sense. Then we could just compare old (2) vs. new (1) and see if it's a significant change. But I see no reason how 0 (interpolating from wrong position) or 3 (huge smoothing, as seen from level 1) could ever make sense. So no need to expose it as a variable (unless it's easier in the code).

Reply:
ok

Reply:
> Let me take it back: we don't need to scan it. I believe that hard-coding it to 1 everywhere is what makes most sense. Then we could just compare old (2) vs. new (1) and see if it's a significant change. But I see no reason how 0 (interpolating from wrong position) or 3 (huge smoothing, as seen from level 1) could ever make sense. So no need to expose it as a variable (unless it's easier in the code).

I agree